Package.swift

```diff
@@ -0,0 +1,17 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Solar",
+    platforms: [.iOS(.v8), .macOS(.v10_15), .watchOS(.v4), .tvOS(.v11)],
+    products: [
+        .library(name: "Solar", type: .dynamic, targets: ["Solar"]),
```

Comment:
If I’m not mistaken, we don’t have to specify a dynamic library here; SPM will choose the right linking based on the containing library or application’s needs.